stacker/actions/build.py

```diff
@@ -89,6 +89,10 @@ def resolve_parameters(parameters, blueprint, context, provider):
             values = value.split(",")
             for v in values:
                 v = v.strip()
+                # Skip parameters not referencing another stack
```

Comment:
I agree. Consolidating that would probably pave the way for the work that's being proposed in #184.